app/src/main/java/com/sopt/clody/ui/theme/Theme.kt

```diff
@@ -1,70 +1,70 @@
 package com.sopt.clody.ui.theme
 
 import android.app.Activity
-import android.os.Build
-import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.material3.MaterialTheme
-import androidx.compose.material3.darkColorScheme
-import androidx.compose.material3.dynamicDarkColorScheme
-import androidx.compose.material3.dynamicLightColorScheme
-import androidx.compose.material3.lightColorScheme
+import androidx.compose.material3.Surface
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.ReadOnlyComposable
 import androidx.compose.runtime.SideEffect
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.graphics.toArgb
-import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.platform.LocalView
 import androidx.core.view.WindowCompat
 
-private val DarkColorScheme = darkColorScheme(
-    primary = Purple80,
-    secondary = PurpleGrey80,
-    tertiary = Pink80
-)
+object ClodyTheme {
+    val colors: ClodyColors
+        @Composable
+        @ReadOnlyComposable
+        get() = LocalClodyColors.current
 
-private val LightColorScheme = lightColorScheme(
-    primary = Purple40,
-    secondary = PurpleGrey40,
-    tertiary = Pink40
-
-    /* Other default colors to override
-    background = Color(0xFFFFFBFE),
-    surface = Color(0xFFFFFBFE),
-    onPrimary = Color.White,
-    onSecondary = Color.White,
-    onTertiary = Color.White,
-    onBackground = Color(0xFF1C1B1F),
-    onSurface = Color(0xFF1C1B1F),
-    */
-)
+    val typography: ClodyTypography
+        @Composable
+        @ReadOnlyComposable
+        get() = LocalTypography.current
+}
 
 @Composable
-fun CLODYTheme(
-    darkTheme: Boolean = isSystemInDarkTheme(),
-    // Dynamic color is available on Android 12+
-    dynamicColor: Boolean = true,
-    content: @Composable () -> Unit
+fun provideClodyColorsAndTypography(
+    colors: ClodyColors,
+    typography: ClodyTypography,
+    content: @Composable () -> Unit,
 ) {
-    val colorScheme = when {
-        dynamicColor && Build.VERSION.SDK_INT >= Build.VERSION_CODES.S -> {
-            val context = LocalContext.current
-            if (darkTheme) dynamicDarkColorScheme(context) else dynamicLightColorScheme(context)
+    val provideColors = remember { colors.copy() }
+    val provideTypography = remember { typography.copy() }
+
+    CompositionLocalProvider(
+        LocalClodyColors provides provideColors,
+        LocalTypography provides provideTypography,
+        content = content,
+    )
+}
```

Comment:
color, type이 immutable이여서 remember로 상태유지를 시킬 필요가 없을 수도 있겠다고 생각됩니다.

```suggestion

    CompositionLocalProvider(
        LocalClodyColors provides colors,
        LocalTypography provides typography,
        content = content,
    )
}
```

Reply:
immutable 어노테이션을 통해서 stable 하다고 해줬으니, 확실히 없어도 될 것 같다는 생각이 듭니다 ! 제거하겠습니다.